spanner/api/Spanner.Samples/CreateInstanceWithoutDefaultBackupSchedules.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_create_instance_without_default_backup_schedule]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Google.Cloud.Spanner.Common.V1;
+using Google.LongRunning;
+using System;
+using System.Threading.Tasks;
+
+public class CreateInstanceWithoutDefaultBackupSchedulesAsyncSample
+{
+  public async Task<Instance> CreateInstanceWithoutDefaultBackupSchedulesAsync(string projectId,
+                                                                               string instanceId) {
+    // Create the InstanceAdminClient instance.
+    InstanceAdminClient instanceAdminClient = await InstanceAdminClient.CreateAsync();
+
+    // Initialize request parameters.
+    Instance instance = new Instance {
```

Comment:
Always on the next line:

```suggestion
    Instance instance = new Instance
    {
```